Ruby/Ruby.sublime-syntax

```diff
@@ -745,6 +745,7 @@ contexts:
 
   method-parameters:
     - meta_content_scope: meta.function.parameters.ruby
+    - include: comments
```

Comment:
The match rules are tested in the order that they appear. So, try to put match rules that pop at the top of the context, and include stuff at the bottom of the context. It doesn't technically matter at this point, because `#` is clearly a different character than `(`, but it's good to have safety nets just in case. I suggest to include the comments context between lines 753 and 754 because of this reasoning. In any case I think this PR is good to merge at this point.